### PR TITLE
Welcome message cosmetic changes

### PR DESCRIPTION
**Pull request by anki-code**
Before:
image

After:
image

### CODE REVIEW

**Comment by AsafFisher** [comment]
`x-on`?

**Comment by anki-code** [comment]
Hi @AsafFisher! What you mean?

**Comment by anki-code** [comment]
Ah, you mean short random line content that xonsh shows after the name. In this PR I'm not changed this. So what you mean? )

**Comment by AsafFisher** [comment]
I take it back, some of the changes look good actually, just need to perfect it (:

**Comment by jnoortheen** [comment]
instead of `x-on steroid` we can show `Python on shell specific steroids` or something similar. There must be a emphasis on it being Python.

**Comment by AsafFisher** [comment]
> instead of `x-on steroid` we can show `Python on shell specific steroids` or something similar. There must be a emphasis on it being Python.

A snake? Inside a conch?

**Comment by daniel-shimon** [comment]
Guys, these sentences (`x-on shell` etc) are randomly generated every time and not a part of the changes in this PR 😁

File: xonsh/xonfig.py [hunk 1]
```diff
@@ -860,12 +860,18 @@ def TAGLINES():
...
     ("{{INTENSE_BLACK}}", "<", "-"),
-    "{{GREEN}}xonfig{{RESET}} tutorial    {{INTENSE_WHITE}}->    Launch the tutorial in "
-    "the browser{{RESET}}",
-    "{{GREEN}}xonfig{{RESET}} web         {{INTENSE_WHITE}}->    Run the configuration "
-    "tool in the browser and claim your shell {{RESET}}",
-    "{{INTENSE_BLACK}}(Note: Run the configuration tool or create a "
-    "{{RED}}~/.xonshrc{{INTENSE_BLACK}} file to suppress the welcome screen)",
+    "",
+    (
+        "{{INTENSE_BLACK}}Create ~/.xonshrc file manually or using xonfig to suppress the welcome screen",
```
Comment (gforsyth) [inline xonsh/xonfig.py:865]:
```suggestion
        "{{INTENSE_BLACK}}Create ~/.xonshrc file manually or use xonfig to suppress the welcome screen",
```

**Comment by melund** [comment]
@anki-code I like the update 👍 

For everyone else who wonder the random puns/jokes are here: https://github.com/xonsh/xonsh/blame/4d7d701c20590cd3e786f4cdf64ea30b4f5c0dac/docs/intro.rst#L9-L47

Fell free to suggest more if you have clever ideas.

**Review by gforsyth — APPROVED** [review]
Thanks for this @anki-code !  Just waiting for CI to finish.

**Pull request merged**